### PR TITLE
Resubmission of PR to remove block by Maartje

### PR DESCRIPTION
**Pull request by remarcmij**

### CODE REVIEW

File: Week1/MAKEME.md [hunk 1]
```diff
@@ -7,108 +7,219 @@ Topics discussed this week:
...
+Please provide feedback on last week's homework from a fellow student as a GitHub issue.
 
-_Deadline Saturday_
+## Step 2: Single Page Application :sweat_drops:
 
-:point_up:
+**_Deadline Thursday_**
 
-## Step 3: SPA :sweat_drops:
+_This homework is more extensive and challenging than previous homework! Please read the instructions below carefully and follow them meticulously. Start this homework as soon as you can and allow time for discussion and questions (slack!)._
```
Comment (mkruijt) [inline Week1/MAKEME.md]: maybe replace "meticulously" with "great attention to detail"

File: Week1/MAKEME.md [hunk 2]
```diff
@@ -7,108 +7,219 @@ Topics discussed this week:
...
-_Deadline Saturday_
+### 2.1 Introduction
 
-You are going to write a SPA (Single Page Application) that uses the [GitHub API](https://developer.github.com/guides/getting-started/).
+You are going to write a _Single Page Application_ (SPA) that uses the [GitHub API](https://developer.github.com/guides/getting-started/).
 
-This application should display information about the available [HYF repositories](https://github.com/hackyourfuture):
+This application should display information about the available [HYF GitHub repositories](https://github.com/hackyourfuture):
 
 - You should be able to select a repository from a list of available repositories.
```
Comment (mkruijt) [inline Week1/MAKEME.md]: maybe we can add "functionalities" or "requirements" on the top of this list?

File: Week1/MAKEME.md [hunk 3]
```diff
@@ -7,108 +7,219 @@ Topics discussed this week:
...
+
+You can find detailed information about the GitHub API by means of the link listed below. However, the documentation is very extensive and not easy to digest. For this homework it is not necessary to study the GitHub API documentation. We provide the link here for completeness.
+
+> GitHub API documentation: https://developer.github.com/v3/
+
+### 2.3 Preparation
+
+You will be working on this same application during the next three weeks. For each week you will need to create a new Git branch, as listed in the Table 1 below.
+
+| Week | Branch | Assignment |
```
Comment (mkruijt) [inline Week1/MAKEME.md]: this is great thank you

File: Week1/MAKEME.md [hunk 4]
```diff
@@ -7,108 +7,219 @@ Topics discussed this week:
...
+
+    ```
+    git checkout -b week1
+    ```
+
+### 2.4 Code Overview
+
+The files that make up the application are located in the `src` folder. It contains the following files:
+
+| Filename         | Description |
```
Comment (mkruijt) [inline Week1/MAKEME.md]: great @remarcmij thank you

File: Week3/MAKEME.md [hunk 5]
```diff
@@ -7,170 +7,120 @@ Topics discussed this week:
...
 ```
 
-## Step 1: Give feedback
+
+## Step 1: Fix requested changes
 
 _Deadline Monday_
 
-Give feedback on Step 3 of last weeks homework. Please provide the feedback in an issue.
+- Fix Requested Changes (if any) on the Pull Request.
```
Comment (mkruijt) [inline Week3/MAKEME.md]: 🙏

File: Week3/MAKEME.md [hunk 6]
```diff
@@ -7,170 +7,120 @@ Topics discussed this week:
...
+#### 2.2.1 async/await
+
+**Instructions:**
+
+1. Refactor all `.then()` and `.catch()` methods with `async`/`await` and `try...catch`.
+
+2. Make sure that your error handling code still works. See the week2 MAKEME on how to force an error response from GitHub.
+
+
+#### 2.2.2 ARIA-compliance
```
Comment (mkruijt) [inline Week3/MAKEME.md]: could you please turn 2.2.2 and 2.2.3 around? And make 2.2.2 the bonus assignment? I'm afraid that this is a lot of homework for one week and that students will not be able to finish all and will leave out 2.2.3

File: README.md [hunk 7]
```diff
@@ -9,7 +9,7 @@ Here you can find course content and homework for the JavaScript3 modules
...
 |2.|• [Event Loop (order of execution)](../../../fundamentals/blob/master/fundamentals/event_loop.md)<br>• [Promises](../../../fundamentals/blob/master/fundamentals/promises.md)|[Reading Week 2](/Week2/README.md)|[Homework Week 2](/Week2/MAKEME.md)|
-|3.|• [try...catch](../../../fundamentals/blob/master/fundamentals/try_catch.md)<br>• [async/await](../../../fundamentals/blob/master/fundamentals/async_await.md)<br>• [Object Oriented Programming and ES6 Classes](../../../fundamentals/blob/master/fundamentals/oop_classes.md)<br>• [The `this` keyword](../../../fundamentals/blob/master/fundamentals/this.md)<br>• call, apply, bind |[Reading Week 3](/Week3/README.md)|[Homework Week 3](/Week3/MAKEME.md)|
+|3.|• [try...catch](../../../fundamentals/blob/master/fundamentals/try_catch.md)<br>• [async/await](../../../fundamentals/blob/master/fundamentals/async_await.md)<br>• [The `this` keyword](../../../fundamentals/blob/master/fundamentals/this.md)<br>• call, apply, bind<BR>• [Object Oriented Programming and ES6 Classes](../../../fundamentals/blob/master/fundamentals/oop_classes.md)|[Reading Week 3](/Week3/README.md)|[Homework Week 3](/Week3/MAKEME.md)|
```
Comment (mkruijt) [inline README.md]: `<br>` in small capitals

File: Week1/MAKEME.md (hunk 1 above)
Comment (remarcmij) [reply Week1/MAKEME.md]: Fixed

File: Week1/MAKEME.md (hunk 2 above)
Comment (remarcmij) [reply Week1/MAKEME.md]: Can you give an example of what you mean?

File: Week3/MAKEME.md (hunk 6 above)
Comment (remarcmij) [reply Week3/MAKEME.md]: 2.2.2 is now ES6 Classes (BONUS)
2.2.3 is now ARIA-compliance

File: README.md (hunk 7 above)
Comment (remarcmij) [reply README.md]: fixed

File: Week3/MAKEME.md [hunk 8]
```diff
@@ -7,170 +7,119 @@ Topics discussed this week:
...
+#### 2.2.1 async/await
+
+**Instructions:**
+
+1. Refactor all `.then()` and `.catch()` methods with `async`/`await` and `try...catch`.
+
+2. Make sure that your error handling code still works. See the week2 MAKEME on how to force an error response from GitHub.
+
+
+#### 2.2.2 ES6 Classes (BONUS)
```
Comment (mkruijt) [inline Week3/MAKEME.md]: I prefer 2.2.3 to be bonus

**Comment by remarcmij** [comment]
Hi Maartje, I closed the previous pull request and created a new one as I was under the assumption that you were (rightly) incommunicado and I wanted to finish this one for class 15 with Laurens.

File: Week1/MAKEME.md (hunk 2 above)
Comment (mkruijt) [reply Week1/MAKEME.md]: yes, so here you give a first indication of the functionalities of what the students should build. So maybe we can add:

functionalities we would like to see in your application:
- this application....
- this application...
- etc
- etc

It is only about the line above the list of functionalities, indicating that this should be the "end" product

File: Week3/MAKEME.md (hunk 8 above)
Comment (remarcmij) [reply Week3/MAKEME.md]: I misunderstood. ARAI compliance is now BONUS instead of ES6 classes.

File: Week1/MAKEME.md (hunk 2 above)
Comment (remarcmij) [reply Week1/MAKEME.md]: Got it! Done.

**Comment by remarcmij** [comment]
Thanks Maartje for taking the time during your holiday to review. Now get back to the business of enjoying your holiday!

**Comment by mkruijt** [comment]
Very welcom Jim. Yes, I will 🌞

**Pull request merged**